### PR TITLE
Create SpringerNature.com.xml

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/SpringerNature.com.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		owa.springernature.com
+
+	Invalid certificate:
+		click.springernature.com
+		editorial-jobs.springernature.com
+		www.search.springernature.com
+		sndigital.springernature.com
+
+	Mixed content:
+		group.springernature.com
```
Comment (J0WI) [inline src/chrome/content/rules/SpringerNature.com.xml]: Is there a specific issue? I wasn't able to find one and the site defaults to https for me.

Comment (Bisaloo) [reply src/chrome/content/rules/SpringerNature.com.xml]: If I don't specify a test url, it's usually on `$`. It seems indeed gone in that case.

**Comment by Bisaloo** [comment]
`id.springernature.com` fails over both HTTP and HTTPS for me.

**Comment by J0WI** [comment]
Try on `http://id.springernature.com/adfs/ls/`

**Comment by Bisaloo** [comment]
Still timing out. Should I add it as a target?

**Comment by J0WI** [comment]
Seems to be HTTPS only.

**Comment by J0WI** [comment]
Sorry, I didn't check the cert chain of `id.springernature.com`

**Pull request merged**